proto/ibc/applications/interchain_accounts/controller/v1/tx.proto

```diff
@@ -26,3 +30,25 @@ message MsgRegisterAccount {
 message MsgRegisterAccountResponse {
   string channel_id = 1 [(gogoproto.moretags) = "yaml:\"channel_id\""];
 }
+
+// MsgSubmitTx defines the payload for MsgSubmitTx
+message MsgSubmitTx {
+  option (gogoproto.equal)           = false;
+  option (gogoproto.goproto_getters) = false;
+
+  string owner         = 1;
+  string connection_id = 2 [(gogoproto.moretags) = "yaml:\"connection_id\""];
+  // Timeout height relative to the current block height.
+  // The timeout is disabled when set to 0.
+  ibc.core.client.v1.Height timeout_height = 3
+      [(gogoproto.moretags) = "yaml:\"timeout_height\"", (gogoproto.nullable) = false];
+  // Timeout timestamp in absolute nanoseconds since unix epoch.
+  // The timeout is disabled when set to 0.
+  uint64 timeout_timestamp = 4 [(gogoproto.moretags) = "yaml:\"timeout_timestamp\""];
+  ibc.applications.interchain_accounts.v1.InterchainAccountPacketData packet_data = 5;
```

Comment:
We should add yaml tags and I think we can probs make this a value type instead of a pointer:
`(gogoproto.nullable) = false`

Reply:
Looks like it needs the yaml tag as well